README.md

```diff
@@ -33,14 +33,19 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
+It is a way of writing code that makes it easier for people, browsers and screen readers, to understand the code, because it is written in a way that describes what it is. Self-referencing html tags
 
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
+display: block; places contents within a block, that can be positioned  within that block. display: inline; places contents within a horizontal line.
```

Comment:
Good answers, just remember that inline does not respect top/bottom properties on the box model